### ISSUE TITLE
Provide Bundle-Version Check

### ISSUE THREAD

**Issue by dstivanov** [openhab/static-code-analysis#197]
As already identified in several places, some of the pull requests that are old are providind bundle version, that is out-of date. For example, the [Energenie Mi|Home binding PR](https://github.com/openhab/openhab2-addons/pull/2278) provides bundle version in the MANIFEST.MF
`Bundle-Version: 2.1.0.qualifier`
while the current release version is already
`Bundle-Version: 2.2.0.qualifier`

The same applies for the version in the pom.xml, which is usually defined as
`<version>2.1.0-SNAPSHOT</version>`

Can you please provide a check, that can be configured for a to check whether the Bundle-Version is specified correctly?

**Comment by martinvw** [comment]
I think that check is there but since the SAT is always based on the code which you are running it has some problems in finding that out I would assume.

**Comment by dstivanov** [comment]
@martinvw You are right - actually there is an option to include Regex for pom.xml and Bundle-version validation in the org.openhab.tools.analysis.checkstyle.PomXmlCheck. This will solve the problem if we set  the regex expressions for the PomXmlCheck.

I discussed with @svilenvul that these regex expressions are currently not set for the SAT nether for the pom.xml version, nor for the bundle-version and therefore cannot be overridden in openhab-addons or smarthome. Fortunately, we can fix it by introducing a general regex expression in the SAT and override it in the openhab-addons and smarthome repositories by altering the ruleset.properties files there.
